packages/@react-aria/grid/src/useGridCell.ts

```diff
@@ -288,3 +447,15 @@ function last(walker: TreeWalker) {
   } while (last);
   return next;
 }
+
+function wrap(walker: TreeWalker) {
+  walker.currentNode = walker.root;
+  let start = walker.firstChild() as FocusableElement;
+  walker.currentNode = walker.root;
+  let end = last(walker);
+  return [start, end];
+}
+
+function isSingleLineTextInput(element: Element) {
```

Comment:
Should this support all [text input types](https://github.com/facebook/react/blob/7c8e5e7ab8bb63de911637892392c5efd8ce1d0f/packages/react-dom-bindings/src/events/isTextInputElement.js)?

Reply:
I imagine so